client/nuxt.config.ts

```diff
@@ -37,6 +38,12 @@ export default defineNuxtConfig({
       }
     }
   },
+  // 
+  ...!isProduction && {
+    alias: {
+      'typesense-instantsearch-adapter': 'typesense-instantsearch-adapter/src/TypesenseInstantsearchAdapter.js',
+    }
+  },
```

Comment:
Like `$production` below there's also a `$development` key that Nuxt config can use to scope